docs/modules/ROOT/pages/contributors/naming-conventions.adoc

```diff
@@ -0,0 +1,126 @@
+= Naming Conventions
+
+This document lists the naming conventions to be applied in the different parts
+of KubeArchive codebase.
+
+== Helm Chart
+
+The Helm Chart templates are located into different directories that match the different
+components of kubearchive:
```

Comment:
Should be "in" rather than "into". "KubeArchive should be capitalized.

```suggestion
The Helm Chart templates are located in different directories that match the different
components of KubeArchive:
```

Reply:
Fixed